src/main/resources/reference.conf

```diff
@@ -41,5 +47,13 @@ kamon {
     # to scale before sending to datadog set "memory-units" to "gb" or "mb" or "kb".
     # Value "b" is equivalent to omitting the setting
     information-unit = "b"
+
+    tag-filters {
```

Comment:
What we are typically doing where filters are needed is to include them in `kamon.util.filters` and then only reference the filter name from the configuration. This made me realize that filters are not explained in the doc so I quickly wrote and published this: http://kamon.io/documentation/1.x/core/basics/utilities/#filters